tests/redux/reducers/search/searchFiltersReducer-test.js

```diff
@@ -133,6 +134,18 @@ describe('searchFiltersReducer', () => {
         });
     });
 
+    describe('UPDATE_TEXT_SEARCH', () => {
```

Comment:
This is a good start, but we'll also need to add tests to `TopFilterBarContainer-test` (showing filter when present, removing from redux when X'd out), create `KeywordContainer-test` tests, and also update the `defaultFilters` object in `testResources/defaultReduxFilters`.